src/components/qaDatatablesContainer/QALinearitySummaryExpandableRows/ProtocolGasDataRows.js

```diff
@@ -0,0 +1,94 @@
+
+import React, { useState, useMemo, useEffect } from "react";
+import { Button } from "@trussworks/react-uswds";
+
+import { getQAProtocolGas } from "../../../utils/api/qaCertificationsAPI";
+import QADataTableRender from "../../QADataTableRender/QADataTableRender";
+import { Preloader } from "@us-epa-camd/easey-design-system";
+
+const tableLabel = 'Protocol Gas'
+
+const ProtocolGasDataRows = ({
```

Comment:
also this component is supposed to be reusable for other test type groups being selected so better to move it under the folder qaDatatablesContainer